node/lib/util/rebase.js

```diff
@@ -448,6 +459,27 @@ Conflict rebasing the submodule ${colors.red(rebaser.path())}.`;
         yield rebaser.finish();
     }));
 
+    // If this was a fast-forward rebase, we need to set the heads of the
```

Comment:
Basically, I have to do the equivalent of a `git submodule update` in each affected submodule or they're sitting on the wrong commit at the end of the rebase. In a non-ffwd rebase, I adjust the submodules as I process the commits.